app/services/customers/fetch-licences.service.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+/**
+ * Fetches the customer licences data needed for the view 'customers/{id}/licences'
+ * @module FetchLicencesService
+ */
+
+const CompanyModel = require('../../models/company.model.js')
+
+/**
+ * Fetches the customer licences data needed for the view 'customers/{id}/licences'
+ *
+ * @param {string} customerId - The customer id for the company
+ *
+ * @returns {Promise<object[]>} the licences for the customer (this will be the licenceDocumentRoles)
+ */
+async function go(customerId) {
+  const company = await _fetch(customerId)
+
+  return company.licenceDocumentRoles
+}
+
+async function _fetch(customerId) {
+  return CompanyModel.query()
```

Comment:
@Cruikshanks i am not sure how we paginate this.

Also this i know this returns the 'licenceDocumentRoles' but the term licences make more sense. So i have left as 'bad' to discuss changing. 

A suggestion for the pagination (assuming i understand the .page limitations) is to get all the licence refs first as a separate query and then build up the licence data from the licence model ?